src/main/java/racinggame/domain/business/RacingGame.java

```diff
@@ -0,0 +1,32 @@
+package racinggame.domain.business;
+
+import racinggame.domain.model.Cars;
+import racinggame.domain.model.RaceCount;
+import racinggame.domain.model.RacingResult;
+
+public record RacingGame(RaceCount raceCount, Cars cars) {
+    
+    public static final int DEFAULT_COUNT = 1;
+    
+    public RacingGame(String carsName) {
+        this(carsName, DEFAULT_COUNT);
+    }
+    
+    public RacingGame(String carsName, int raceCount) {
+        this(new RaceCount(raceCount), new Cars(carsName));
+    }
+    
+    public void race() {
+        this.cars.carsRace();
+        this.raceCount.decreaseCount();
+    }
+    
+    public RacingResult raceResult() {
+        return new RacingResult(this.cars);
+    }
+    
+    public boolean isEnd() {
+        return !raceCount.isZeroOrLower();
```

Comment:
```suggestion
        return raceCount.isPositive();
```
부정문보다 메서드 추가해 구현하는 것을 추천